plugins/micromega/mutils.ml

```diff
@@ -420,6 +427,52 @@ let command exe_path args vl =
              stdout_read; stdout_write;
              stderr_read; stderr_write])
 
+(** Hashing utilities - borrowed from OCaml hash.c *)
+
+module Hash =
+  struct
+
+    module Mc = Micromega
+
+    let rotl32 x n = (x lsl n) lor (x lsr (32 - n))
+
+    let mix h d =
```

Comment:
This could be `Hashset.Combine.combine`.

Reply:
Thanks. Done.